import_export/forms.py

```diff
@@ -26,6 +27,11 @@ def __init__(self, import_formats, *args, **kwargs):
 
         self.fields['input_format'].choices = choices
 
+        if hasattr(settings, 'IMPORT_DEFAULT'):
+            for i, c in enumerate(choices):
+                if c[1] == settings.IMPORT_DEFAULT.lower():
```

Comment:
I don't think there's anything that forces the import format title to be lowercase, so I don't think should be calling `.lower()` on the `IMPORT_DEFAULT` setting.

Reply:
In fact, there should be a .lower() on both sides of the comparison... to deal with both/all cases.

Reply:
That could work, though still forces consideration of case in this one particular case.  So that it's usable elsewhere, I wonder if the best way might be to add a new method to the `Format` class that just returns `self.get_title().lower()` and we use that to test instead?

Reply:
I'm a bit confused! it only casts the extension to lower case... to deal with the possibility that someone as saved their file as file.XLS, etc. Are you clear on that? If so then my confusion continues...

Reply:
There should be no lower() at all. Values are from constants in import_export or provided by the programmer, also this new option will be provided by the programmer, the extension of the file is never checked here, or I'm missing something :)